composables/api/useConfirmAccount/index.ts

```diff
@@ -15,6 +15,7 @@ export async function useConfirmAccount(
     baseUrl: useRuntimeConfig().public.backendBaseUrl,
   })
 
+  // eslint-disable-next-line @typescript-eslint/no-invalid-void-type
```

Comment:
I feel like we can get around having to do this by updating the composable, but this is fine for now

Reply:
Agreed. Make an issue for it.